Example/Messaging/Tests/FIRMessagingExtensionHelperTest.m

```diff
@@ -70,9 +70,11 @@ - (void)testModifyNotificationWithValidPayloadData {
   OCMVerify([_mockExtensionHelper loadAttachmentForURL:[OCMArg any]
                                      completionHandler:[OCMArg any]]);
   [self waitForExpectationsWithTimeout:1.0 handler:nil];
```

Comment:
I reproduced the timeout failure once in four runs running locally.

Reply:
I'm still seeing a failure here.  Is there an issue with _mockExtensionHelper being used by multiple tests running in parallel?